packages/editor/src/components/autocompleters/block.js

```diff
@@ -54,7 +54,7 @@ export function createBlockCompleter( {
 } = {} ) {
 	return {
 		name: 'blocks',
-		className: 'editor-autocompleters__block',
+		className: 'block-editor-autocompleters__block',
```

Comment:
Should I have gone with `block-editor`, `editor` (changing the stylesheet instead), or both?

Reply:
It seems like we should go with `editor` as we're declaring these in the `editor` module. Not sure why these classNames were changed.